codebuild/spec/buildspec_sidetrail.yml

```diff
@@ -35,7 +35,13 @@ phases:
   build:
     commands:
       - printenv
-      - codebuild/bin/run_sidetrail.sh /sidetrail-install-dir ${CODEBUILD_SRC_DIR}
+      - |
+        if [ -d "third-party-src" ]; then
+          cd third-party-src
+          ./codebuild/bin/run_sidetrail.sh /sidetrail-install-dir ${CODEBUILD_SRC_DIR}/third-party-src;
```

Comment:
Throughout this change, we use tha path `./codebuild/bin/`pretty often, can we store this in a global path variable (`codebuild_bin_path:  or $CODEBUILD_BIN_DIR` and use it?

Reply:
Sure, [globals](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html) should work.